infra/Pos/DHT/Real/Real.hs

```diff
@@ -38,6 +39,7 @@ import           Pos.DHT.Real.Types        (DHTHandle, KademliaDHT (..),
 import           Pos.Util.TimeLimit        (runWithRandomIntervals')
 import           Pos.Util.TimeWarp         (NetworkAddress)
 
+
```

Comment:
We don't usually have 2 spaces between imports and code, perhaps this was a mistake?